Webhook docs with code examples for signatures calculation (#54)
e2b-dev/docs: docs/sandbox/lifecycle-events-webhooks.mdx

}
```

# Webhook verification
To ensure the authenticity of webhook requests, each request includes a signature in the `e2b-signature` header.
You can verify the signature using the signature secret you provided when registering the webhook.

> This helps confirm that the request originated from E2B andhas not been tampered with.

Reviewer: ```suggestion
This confirms that the request originated from E2B and has not been tampered with.
```
